### PR TITLE
App bugs

### PR DESCRIPTION
**Pull request by perryr16**
approved is now a column of pet_apps instead of apps

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -172,3 +172,10 @@
 .user-message {
   margin-top:40px;
 }
+.section-right {
+  float: left;
+  margin-left: 5vw;
+}
```
Comment (perryr16) [inline app/assets/stylesheets/application.css:178]: styling

File: app/controllers/apps_controller.rb [hunk 2]
```diff
@@ -17,7 +21,7 @@ def create
 
     if @app.save
       params[:pet_ids].each do |id|
-        PetApp.create(pet_id: id, app: @app)
+        PetApp.create(pet_id: id, app_id: @app.id, approved: false)
```
Comment (perryr16) [inline app/controllers/apps_controller.rb:24]: not sure how we were getting away with this old code

File: app/models/pet.rb [hunk 3]
```diff
@@ -31,8 +31,20 @@ def self.pet_list(params)
   end
 
   def on_hold_for
-    if !adoptable
-      apps.where(approved: :true).pluck(:name).join
+    # binding.pry
+    # if !adoptable
+    #   apps.where(approved: :true).pluck(:name).join
+    #
+    # end
```
Comment (perryr16) [inline app/models/pet.rb:38]: delete this text

File: app/models/pet.rb [hunk 4]
```diff
@@ -31,8 +31,20 @@ def self.pet_list(params)
...
+    pet_app = PetApp.where(pet_id: id).first
+    if pet_app.approved
+      app_name = App.find(pet_app.app_id).name
+    else
+      nil
     end
   end
+
+  def approved
+    PetApp.where(pet_id: id).where(approved: :true).exists?
```
Comment (perryr16) [inline app/models/pet.rb:48]: view conditional

File: app/assets/stylesheets/application.css (hunk 1 above)
Comment (muydanny) [reply app/assets/stylesheets/application.css:178]: love styling

File: app/controllers/apps_controller.rb (hunk 2 above)
Comment (muydanny) [reply app/controllers/apps_controller.rb:24]: Not sure either but its working...for the love of god its working

**Pull request merged**